assets/embauche.html

```diff
@@ -107,177 +107,179 @@
 	<details id="taxes" class="js-only">
 		<summary>Cotisations détaillées</summary>
 
-		<table>
-			<caption>Cotisations employeur</caption>
+		<div class="tables">
```

Comment:
Yeah, sure! That's why you should use the same state-based selector as the polyfill uses for setting `display: block`:  `details.▼ > *`  ;)